src/routes/components/dialog/+page.svelte

```diff
@@ -59,93 +59,55 @@
 </script>
 
 <QDocs componentDocs={QDialogDocs}>
-  <QDialog
-    slot="display"
-    button
-    buttonLabel="Open cookie settings"
-    modal
-    persistent
-    bind:value={exampleDialog.value}
-    bind:this={exampleDialog.el}
-  >
-    <DialogContent dialogEl={exampleDialog.el} />
-  </QDialog>
+  <svelte:fragment slot="display">
+    <QBtn onclick={exampleDialog.el?.show}>Open cookie settings</QBtn>
+
+    <QDialog modal persistent bind:value={exampleDialog.value} bind:this={exampleDialog.el}>
```

Comment:
Attributes should be on new lines, no?

Reply:
If prettier decided that nope, I guess it's "nope" lol

Reply:
The line is 93 characters, less than 100 so prettier decides to keep it as a line